src/main/kotlin/no/nav/syfo/meroppfolging/SenOppfolgingSvarProducer.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.syfo.meroppfolging
+
+import no.nav.syfo.meroppfolging.model.SenOppfolgingSvar
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import java.util.*
```

Comment:
Bør vi finne en måte for å unngå `import ... .*`?

Reply:
I dette repoet er det tillatt med star import: https://github.com/navikt/isyfomock/blob/ecc0028e2936e8c17fff47d32624dd618730ae01/.editorconfig#L2

Dette er isyfo sitt repo, så jeg ønsker ikke å endre oppsettet på det.